docs/source/app-dev/bindings-java/index.rst

```diff
@@ -84,53 +75,6 @@ The ``LedgerView`` is updated every time:
 For instance, if an incoming transaction is received with a create event for a contract that is relevant
 for the application, the application ``LedgerView`` is updated to contain that contract too.
 
-Writing automations: Bot
```

Comment:
Why are we removing this section? I thought we were only removing the example and references to that, not the whole documentation on the reactive components.

Reply:
Sorry, that was a misunderstanding. I pulled the documentation itself back in, but the reference in the example page is now gone.